### ISSUE TITLE
"looks like HTML" warning broken

### ISSUE THREAD

**Issue by jwilk** [wking/rss2email#83]
In `rss2email/feed.py` there's this code to warn when the server sent you HTML:

``` python
http_headers = parsed.get('headers', {})
...
if 'html' in http_headers.get('content-type', 'rss'):
    _LOG.warning('looks like HTML: {}'.format(self))
    warned = True
```

But at least with feedparser v5.1.3, `http_headers` is a normal dict with keys that were _not_ lower-cased. So this check won't trigger most of the time.

**Comment by Ekleog** [comment]
Hello,

This repository has been deprecated for a few years now, and has been replaced by https://github.com/rss2email/rss2email .

If this issue is still relevant to you, and not fixed with v3.12.2, could you please reopen the issue there?

Cheers,
  Leo

**Comment by jwilk** [comment]
Closing in favor of <https://github.com/rss2email/rss2email/issues/160>.
